ReeldIn/settings.py

```diff
@@ -143,6 +140,12 @@
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
 STATIC_URL = "static/"
+STATIC_ROOT = BASE_DIR / "static"
```

Comment:
this won't mess with anything else down the line right?

Reply:
No this actually has to do with when we set up the website, we have to do some collectstatic command or something which uses the root (see https://docs.djangoproject.com/en/5.0/howto/static-files/deployment/ for more info)